extensions/resteasy-classic/resteasy/deployment/src/main/java/io/quarkus/resteasy/deployment/ResteasyStandaloneBuildStep.java

```diff
@@ -158,6 +160,29 @@ public void boot(ShutdownContextBuildItem shutdown,
         recorder.start(shutdown, requireVirtual.isPresent());
     }
 
+    private static boolean foundOtherCustomExMapper(IndexView index) {
+        // check if user specified custom exception mapper that is not an exception mapper for one of auth exceptions
+        // builtin exception mappers are not supposed to be indexed, therefore this should only detect application one's
+        for (var implementor : index.getAllKnownImplementors(EXCEPTION_MAPPER)) {
+            for (Type interfaceType : implementor.interfaceTypes()) {
+                if (EXCEPTION_MAPPER.equals(interfaceType.name())) {
+                    final String mapperExSignature = interfaceType.asParameterizedType().arguments().get(0).name().toString();
+                    if (AuthenticationCompletionException.class.getName().equals(mapperExSignature)) {
```

Comment:
If you'd like you might try keeping these names in the static set, or do some switch, but this is not really verbose so it is up to you

Reply:
I actually think we can drop this one, I have only discovered runtime way to detect exact exception mapper for an exception later. Going this extra mile could make sense in Quarkus REST (where I didn't do it because I wasn't sure what I do was reliable), but no in the RESTEasy Classic. I'll drop it.